src/pages/settings/Profile/ProfilePage.js

```diff
@@ -346,9 +346,11 @@ class ProfilePage extends Component {
                             />
                         </View>
                         <CheckboxWithLabel
+                            inputID="shouldSetTimezoneAutomatically"
```

Comment:
NAB (Not a blocker): `shouldSetTimezoneAutomatically` -> `setTimezoneAutomatically`

Reply:
Refactored the inputID